Silicon/TigerlakePkg/Include/Register/SaRegs.h

```diff
@@ -0,0 +1,1835 @@
+/** @file

```

Comment:
A lot of definitions here. Do we need all of them?

Reply:
I removed all the unused parameters.